Doc/library/sqlite3.rst

```diff
@@ -2266,11 +2333,62 @@ Explanation
 
 .. _sqlite3-controlling-transactions:
 
-Transaction control
-^^^^^^^^^^^^^^^^^^^
+Transaction control via the ``autocommit`` attribute
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the :attr:`~Connection.autocommit` attribute to select
+transaction control mode.
+This attribute should preferrably be set using the *autocommit* parameter
+of :func:`connect`.
+
+Currently, *autocommit* defaults to :const:`DEPRECATED_TRANSACTION_CONTROL`.
+This causes transaction control to be selected using the
+:attr:`~Connection.isolation_level` attribute.
+See :ref:`sqlite3-deprecated-transaction-control` for more information.
+
+Starting in Python 3.14, *autocommit* will default to ``False``,
+which will imply :pep:`249`-compliant transaction control, meaning:
+
+* :mod:`!sqlite3` ensures that a transaction is always open,
+  meaning :meth:`~Connection.commit` and :meth:`~Connection.rollback`
+  will implicitly open new transactions immediately after closing
+  the pending transaction.
+  :mod:`!sqlite3` uses ``BEGIN DEFERRED`` statements when opening transactions.
+* Transactions should be committed explicitly
+  using :meth:`!~Connection.commit`.
+* Transactions should be rolled back explicitly
+  using :meth:`!~Connection.rollback`.
+* An implicit rollback is performed if the database is closed with
+  pending changes.
+
+Set *autocommit* to ``True`` to enable SQLite's `autocommit mode`_.
+In this mode, :meth:`~Connection.commit()` and :meth:`~Connection.rollback()`
+have no effect.
+
+.. note::
+
+   :attr:`~Connection.autocommit` and SQLite's autocommit mode are not linked.
+   For example,
+   explicitly issuing a ``BEGIN`` statement when ``autocommit=True``,
+   will not change the value of the *autocommit* attribute.
 
-The :mod:`!sqlite3` module does not adhere to the transaction handling recommended
-by :pep:`249`.
+   Use :attr:`~Connection.in_transaction` to query the low-level SQLite
```

Comment:
```suggestion
   Use :attr:`Connection.in_transaction` to query the low-level SQLite
```

IMO, its useful to be clear that this is an attribute of `Connection`, rather than anything else, without having to click through the link.